js/base/Exchange.js

```diff
@@ -1394,6 +1394,16 @@ module.exports = class Exchange {
         return indexed ? this.indexBy (result, 'currency') : result;
     }
 
+    safeDepositAddress (depositAddress) {
+        return this.extend({
```

Comment:
If possible, please use consistent spacing here `this.extend({` > `this.extend ({` to keep it uniform across the file.